src/helpers/is-user-in-team.ts

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2023 Expedia, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import * as core from '@actions/core';
+import { HelperInputs } from '../types/generated';
+import { context } from '@actions/github';
+import { octokitRequest } from '../octokit';
+import { OctokitResponse } from '@octokit/types';
+
+export class IsUserInTeam extends HelperInputs {
+  login = '';
+  team = '';
+}
+
+export const isUserInTeam = async ({ login = context.actor, team }: IsUserInTeam) => {
+  if (!team) {
+    core.error('Must supply team');
+  }
+  const response: OctokitResponse<Record<string, string | number | boolean>[]> = await octokitRequest(
+    'GET /orgs/{org}/teams/{team}/members',
```

Comment:
Can we not use octokit for this?

Reply:
annoyingly, octokit doesn't have a wrapper for this (unless you can find one-- i couldn't). the closest thing i see is `octokit.rest.teams.getMembershipForUserInOrg` which 404's if the user isn't in the team or `octokit.rest.teams.getByName` which doesn't include members in the response

Reply:
Try teams.listMembersInOrg, which is what we use in a different helper